catalog/nsili/catalog-nsili-endpoint/src/main/java/com/connexta/alliance/nsili/endpoint/LibraryImpl.java

```diff
@@ -78,12 +108,15 @@ public LibraryManager get_manager(String manager_type, AccessCriteria access_cri
         org.omg.CORBA.Object obj;
 
         if (manager_type.equals(NsiliManagerType.CATALOG_MGR.getSpecName())) {
-            CatalogMgrImpl catalogMgr = new CatalogMgrImpl(poa);
+            CatalogMgrImpl catalogMgr = new CatalogMgrImpl(poa, filterBuilder);
+            catalogMgr.setCatalogFramework(catalogFramework);
+            catalogMgr.setGuestSubject(guestSubject);
             try {
                 poa.activate_object_with_id(manager_type.getBytes(Charset.forName(ENCODING)),
                         catalogMgr);
             } catch (ServantAlreadyActive | ObjectAlreadyActive | WrongPolicy e) {
-                LOGGER.error("Error activating CatalogMgr", e);
+                LOGGER.info(String.format("Error activating CatalogMgr: %s",
```

Comment:
Can't we do `LOGGER.info("Error activating CatalogMgr: {}", e.getLocalizedMessage())`?